dev/archery/archery/release.py

```diff
@@ -85,11 +84,9 @@ def number(self):
 
 class Jira(JIRA):
 
-    def __init__(self, user=None, password=None,
+    def __init__(self, token=None,
```

Comment:
This doesn't seem to be used anymore, we should remove it from the signature too.